openstack/neutron/templates/etc/_ironic_neutron_agent.ini.tpl

```diff
@@ -10,6 +10,6 @@ auth_url = {{.Values.global.keystone_api_endpoint_protocol_internal | default "h
 project_name = {{ default "service" .Values.global.keystone_service_project }}
 project_domain_name = {{ default "Default" .Values.global.keystone_service_domain }}
 username = {{ $user }}
-password = {{ coalesce .Values.global.ironicServicePassword .Values.global.ironic_service_password  (tuple . $user | include "identity.password_for_user")  | replace "$" "$$" }}
+password = {{ coalesce .Values.global.ironicServicePassword .Values.global.ironic_service_password }}
```

Comment:
As far as I see, only `ironic_service_password` is used in the globals, can we use that solely? Also, please use required macro, e.g. `password = {{ required ".Values.global.ironic_service_password required!" .Values.global.ironic_service_password }}`. Thanks!

Reply:
i wasnt aware which parameter is used for password, hence i did not specify required only for this PR. 

i dont think we can get rid of ironic service password in the globals, since its used in other ironic services https://github.com/sapcc/helm-charts/pull/2348/files